### DISCUSSION TITLE
Abrindo outra página ao clicar na div que do Jumbotron

### DISCUSSION THREAD

**Discussion by jcvilanova** [frontendbr/forum#2290] [Perguntas e Respostas] [answered]
Galera, nesse jumbotron eu coloquei uma imagem de fundo e esse h1.  o H1 tem um link para outra página, no entanto, eu quero retirar o h1 e deixar apenas a imagem, porem, quero que ela continue linkando as duas páginas. Como posso fazer isso ? 

```html
<div class="jumbotron p-4 text-white" style="background-image: url('<?php echo URL . 'assets/imagens/artigo/' . $imagem; ?>');">
  <div class="col-md-6 px-0">
    <h1 class="display-4 fst-italic"><?php echo $titulo; ?></h1>
  </div>
</div>
```

**Comment by jcvilanova** [comment]
tentei colocar o codigo aqui para verem mas nao aparefce

**Reply by felquis** [reply]
É assim:
\```html
  // código aqui
\```

**Comment by felquis** [comment]
É isso que você quer?

Como você disse que não quer o h1 ficaria +- dessa forma. 
```html
<a class="jumbotron p-4 text-white w-64 h-64 block" style="background-image: url('<?php echo URL . 'assets/imagens/artigo/' . $imagem; ?>');">
  <h1 class="sr-only"><?php echo $titulo; ?></h1>
</a>
```

Coloquei um tamanho width e height para o link e coloquei block pois por padrão ancoras são display inline. E precisa de display block ou inline-block para poder usar width e height em elementos inline.

**Comment by jcvilanova** [comment]
ess h1 deixaria de existir. ficaria apenas a imagem de fundo no jumbotron. Ai, clicando em qualquer parte do jumbotron abriria a pagina correspondente. no seu exemplo ai onde está o link ?

**Reply by jcvilanova** [reply]
coloquei o seu código e nao funcionou. Ao passar o mouse nada acontece

  <div class="jumbotron p-4 text-white w-64 h-64 block" style="background-image: url('<?php echo URL . 'assets/imagens/artigo/' . $imagem; ?>');">

**Reply by felquis** [reply]
Ha, esqueci de trocar a DIV por um A.

**Reply by felquis** [reply]
Editei com o código correto.

**Reply by jcvilanova** [reply]
não está aparecendo aqui pra mim o seu código

**Reply by felquis** [reply]
aqui https://github.com/frontendbr/forum/discussions/2290#discussioncomment-2695209

**Reply by jcvilanova** [reply]
como faço para te mandar a captura de tela por aqui? depois que troquei o <div> por <a> ficou assim, quero mandar as imagens de antes e depois pra voce ver o que aconteceu

**Reply by jcvilanova** [reply]
![Italia](https://user-images.githubusercontent.com/90116702/166981708-e4eb3698-5273-4a1d-816e-071bae369506.PNG)

estava assim

**Comment by jcvilanova** [comment]
depois que fiz o que voce mandou ficou assim
![Italia 2](https://user-images.githubusercontent.com/90116702/166981827-650e0fc9-47e3-4dc4-882a-f4d3f4ae55ab.PNG)

**Comment by jcvilanova** [comment]
ai eu nao entendi

**Comment by jcvilanova** [comment, marked as answer]
Consegui resolver atraves do javascript
